### PR TITLE
murdock: Add a few more metrics

### PR DESCRIPTION
**Pull request by bergzand**
- Number of workers
- Websockets active
- Push events counter
- Pull events counter

Depends on #22

### CODE REVIEW

**Comment by bergzand** [comment]
rebased!

File: murdock/murdock.py [hunk 1]
```diff
@@ -497,6 +520,11 @@ def handle_ref(ref: str, rules: List[str]) -> bool:
         )
 
     async def handle_push_event(self, event: dict):
+        # catchall name
+        metric_repo_name = "undefined"
+        if "repository" in event and "full_name" in event["repository"]:
+            metric_repo_name = event["repository"]["full_name"]
+        self.webhook_push_event_counter.labels(repository=metric_repo_name).inc()
```
Comment (aabadie) [inline murdock/murdock.py:527]: With this event you will get all push events on the main repo, e.g push to master (merge) and new tags. For RIOT-OS this also contains all pushes the release branches. Is it what you want ?

Comment (bergzand) [reply murdock/murdock.py:527]: I couldn't find an easy way to distinguish these push events. Do you know if the event data contains something similar like the `action` in the pull event data?

Comment (bergzand) [reply murdock/murdock.py:527]: Having a label that distinguishes merges from tags and other types would be nice to have

Comment (bergzand) [reply murdock/murdock.py:527]: If I understand the docs, we should be able to distinguish merges, tags and such by the ref property, but I'm not willing to implement that without some example payloads.

Comment (aabadie) [reply murdock/murdock.py:527]: The ref contains information if it's a tag or a branch, see https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push:

```
The full git ref that was pushed. Example: refs/heads/main or refs/tags/v3.14.1.
```

Comment (bergzand) [reply murdock/murdock.py]: I've added a label that contains either "commit" or "tag".

**Review by aabadie — APPROVED** [review]

**Pull request merged**